library/cohesity_policy.py

```diff
@@ -0,0 +1,378 @@
+# !/usr/bin/python
+# Copyright (c) 2019 Cohesity Inc
+# Apache License Version 2.0
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+
+from ansible.module_utils.basic import AnsibleModule
+from cohesity_management_sdk.cohesity_client import CohesityClient
+from cohesity_management_sdk.exceptions.api_exception import APIException
+from cohesity_management_sdk.models.protection_policy_request import ProtectionPolicyRequest
+from cohesity_management_sdk.models.blackout_period import BlackoutPeriod
+from cohesity_management_sdk.models.time_of_day import TimeOfDay
+from cohesity_management_sdk.models.scheduling_policy import SchedulingPolicy
+from cohesity_management_sdk.models.daily_schedule import DailySchedule
+from cohesity_management_sdk.models.monthly_schedule import MonthlySchedule
+from cohesity_management_sdk.models.continuous_schedule import ContinuousSchedule
+from cohesity_management_sdk.models.extended_retention_policy import ExtendedRetentionPolicy
+from cohesity_management_sdk.models.snapshot_archival_copy_policy import SnapshotArchivalCopyPolicy
+from cohesity_management_sdk.models.archival_external_target import ArchivalExternalTarget
+
+try:
+    from module_utils.storage.cohesity.cohesity_utilities import cohesity_common_argument_spec,\
+        raise__cohesity_exception__handler
+except Exception:
+    from ansible.module_utils.storage.cohesity.cohesity_utilities import cohesity_common_argument_spec,\
+        raise__cohesity_exception__handler
+
+
+cohesity_client = None
```

Comment:
Why is this init to None here?

Reply:
cohesity_client is a global variable, Is there a way we can declare a global variable without initializing in python?, It can be initialized to any value, since its a null object initialized it to None

Reply:
Initialize it globally then?